config/checkstyle_resources_checks.xml

```diff
@@ -33,6 +34,34 @@
                       ''Input*.java'' or ''package-info.java'' or ''*.properties''."/>
     </module>
 
+    <!-- Miscellaneous -->
+    <module name="NewlineAtEndOfFile" />
+    <module name="RegexpSingleline">
+      <property name="id" value="lineLength"/>
+      <!-- catch lines above 100 symbols -->
+      <property name="format"
+               value="^(?!(.*href=|.*http(s)?:|import |(.* )?package |.* files=|.*\.dtd)).{101,}$"/>
+      <property name="message" value="Line should not be longer than 100 symbols"/>
+    </module>
+
+    <!-- check the number of testCases -->
+    <module name="RegexpSingleline">
```

Comment:
Something is wrong with this rule. A build on my PC succeeded after I deleted one test from `src/it/java/org/checkstyle/suppressionxpathfilter/XpathRegressionAnonInnerLengthTest.java`
On master branch it failed as expected.